src/Uno.UI.RemoteControl.Host/Program.cs

```diff
@@ -81,14 +87,23 @@ static async Task Main(string[] args)
 					services.AddSingleton<IIdeChannel, IdeChannelServer>();
 				});
 
+			Debugger.Launch();
+
 			if (solution is not null)
 			{
 				// For backward compatibility, we allow to not have a solution file specified.
 				builder.ConfigureAddIns(solution);
 			}
+			else
+			{
+				typeof(Program).Log().Log(LogLevel.Warning, "No solution file specified, add-ins will not be loaded.");
```

Comment:
should we add additional context in the log message (e.g. potential impacts of not having a solution file) to inform the user ?

Reply:
```suggestion
				typeof(Program).Log().Log(LogLevel.Warning, "No solution file specified, add-ins will not be loaded which means that you won't be able to use any of the uno-studio features. Usually this indicates that your version of uno's IDE extension is too old.");
```